bower.json

```diff
@@ -17,6 +17,7 @@
   ],
   "dependencies": {
     "angular": "~1.3.8",
-    "angular-mocks": "~1.3.8"
+    "angular-mocks": "~1.3.8",
```

Comment:
This is something I really dislike about JSON: When you append a line, you need to change the previous line. CSON does a nice job on that.

(No need to change anything, I'm just sharing my thoughts)